Frontend/js/src/sessionService.js

```diff
@@ -0,0 +1,61 @@
+/*global: window*/
```

Comment:
I'd say it's more like key-value (or parameter) service. There are no sessions, what do you think?